### DISCUSSION TITLE
doc/java/guide/server-common-report/%E6%8A%A5%E8%A1%A8%E7%BB%84%E4%BB%B6%E6%8E%A5%E5%85%A5%E6%B5%81%E7%A8%8B_V1.0.0

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#930] [Announcements]
# doc/java/guide/server-common-report/%E6%8A%A5%E8%A1%A8%E7%BB%84%E4%BB%B6%E6%8E%A5%E5%85%A5%E6%B5%81%E7%A8%8B_V1.0.0

公共报表组件接入流程 文档版本号 SDK版本号 更新时间 更新人 ---------- -------------- ------------------- ------ V1.0.0 1.0.0-SNAPSHOT 2024-09-27 15:30:00 康永 一、组件描述 1.1、服务解决的问题 * 复杂报表实时调用超时，任务堵塞 * 针对复杂报表提...

https://demo-lvzhou.youmatech.com/doc/java/guide/server-common-report/%E6%8A%A5%E8%A1%A8%E7%BB%84%E4%BB%B6%E6%8E%A5%E5%85%A5%E6%B5%81%E7%A8%8B_V1.0.0.html

**Comment by kzdwts** [comment]
great
